projects/netgrif-components/src/lib/header/header.component.html

```
@@ -43,8 +43,8 @@
                     <mat-icon>more_vert</mat-icon>
                 </button>
                 <mat-menu  #headerMenu="matMenu" class="mat-menu-border-radius mat-menu-padding">
-                    <div *ngIf="headerService.headerState.mode !== headerModeEnum.EDIT && showSortButton">
-                        <button mat-menu-item
+                    <div *ngIf="headerService.headerState.mode !== headerModeEnum.EDIT">
+                        <button *ngIf="showSortButton" mat-menu-item
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)